workers/data_refinery_workers/processors/array_express.py

```diff
@@ -141,7 +141,7 @@ def _create_result_objects(job_context: Dict) -> Dict:
         computed_file.calculate_sha1()
         computed_file.calculate_size()
         computed_file.result = result
-        computed_file.sync_to_s3(S3_BUCKET_NAME, computed_file.sha1 + "_" + computed_file.filename)
+        # computed_file.sync_to_s3(S3_BUCKET_NAME, computed_file.sha1 + "_" + computed_file.filename)
```

Comment:
No file sync any more?

Reply:
File syncing is currently suspended while we a) figure out how we want to provide the data to the end-user (this will require a "smasher" Job and b) an "Uploader" job to move the data to S3 on a lower-cost instance type than the big machines used to crunch.